cdk-projects/bash_scripts/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM amazonlinux:2 as bootstrap
```

Comment:
You have added duplicate Dockerfiles (and some other files), so make sure only the relevant files are added